app/content/tiles/jadeInteriorLight.ts

```diff
@@ -1,20 +1,42 @@
 import {
-    southernWallBehavior,
-    topRightWall,
-    topLeftWall,
-    bottomLeftWall,
-    bottomRightWall,
-    ceilingBehavior,
-    bottomCeilingBehavior,
-    topLeftCeiling,
-    topRightCeiling,
-    bottomLeftCeiling,
-    bottomRightCeiling,
+    baseCeilingBehavior,
 } from 'app/content/tiles/constants';
 
+import {
+    BITMAP_BOTTOM,
+    BITMAP_BOTTOM_LEFT,
+    BITMAP_TOP_LEFT,
+    BITMAP_TOP_RIGHT,
+    BITMAP_BOTTOM_RIGHT,
+    BITMAP_BOTTOM_LEFT_8,
+    BITMAP_BOTTOM_RIGHT_8,
+    BITMAP_TOP_LEFT_8_STRIP,
+    BITMAP_TOP_RIGHT_8_STRIP,
+} from 'app/content/bitMasks';
+
 
 import { requireFrame } from 'app/utils/packedImages';
 
+const southernWallBehavior: TileBehaviors = {
+    solid: true,
+    // Wall appear behind the player except over doorways.
+    defaultLayer: 'field',
+    isSouthernWall: true,
+    linkedOffset: 57,
+}
+const ceilingBehavior: TileBehaviors = { ...baseCeilingBehavior, solid: true, linkedOffset: 57};
```

Comment:
This is a good idea, but in the long run it will probably be convenient to keep inheriting all the behaviors from the constants file. Can we change all of these local constants to work like this instead?
`const linkedCeilingBehavior: TileBehaviors = { ...ceilingBehavior, linkedOffset: 57}`;

I think this gives us the best of both worlds.